azafea/event_processors/endless/activation/v1/handler.py

```diff
@@ -32,13 +32,9 @@ class Activation(Base):
     vendor = Column(Unicode, nullable=False)
     product = Column(Unicode, nullable=False)
     release = Column(Unicode, nullable=False)
-    serial = Column(Unicode)
```

Comment:
I think we need to make `from_serialized` ignore the `"serial"` and `"mac_hash"` fields by deleting them from `record` before attempting to construct an instance of `Activation`.

That method implicitly assumes that the incoming record only includes keys which are defined as fields on this class. If we were to merge this PR and deploy it, it would fail for all incoming activations because they will still contain `serial` and `mac_hash`. (We can perhaps also make `eos-activation-server` strip them out of incoming requests before putting them into the redis queue.)

I had a quick look at the integration tests and none of them include the `serial` or `mac_hash` fields. Please add an integration test that verifies that these fields are ignored if present in the inccoming request. I think it would be as simple as adding these two fields to the dict in `test_activation_parsing.py`.

Reply:
Ok, I modified `from_serialized` and the test in `test_activation_parsing.` 

But then after testing, sqlalchemy is smart enough to ignore `serial` and `mac_hash` if they are in the record but not in the model fields, I had no error on the tests. But let's remove the fields  from record received anyway!

Reply:
That is quite surprising! I would normally expect passing an unexpected kwarg to be fatal. Is that sqlalchemy behaviour documented anywhere?

Reply:
Or is it specific to the models in Azafea?

If that's so I don't think those fields need to be explicitly dropped. Just the test is fine IMO.

Reply:
Yes, I check, and it's specific to azafea! 
in `azafea.models` on the class `BaseModel` : 
```
def __init__(self, **kwargs: Any):
        columns = inspect(self.__class__).attrs

        for k, v in kwargs.items():
            if k in columns:
                setattr(self, k, v)
```